### PR TITLE
Read database from env var

### PR DESCRIPTION
**Pull request by janopae**
Added support for reading the DSN from a MYSQL_DSN environment variable

### CODE REVIEW

File: src/Webfactory/Slimdump/SlimdumpCommand.php [hunk 1]
```diff
@@ -35,6 +35,11 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $dsn = $input->getArgument('dsn');
+
+        if ($dsn === '-'){
```
Comment (mpdude) [inline src/Webfactory/Slimdump/SlimdumpCommand.php]: Space missing before { (coding standards)

**Comment by janopae** [comment]
No, I did not change it intentionally. I thought, while the composer.json is a configuration file, composer.lock might be an automatic generated file that might got updated after running composer update.

**Comment by mpdude** [comment]
As that change has nothing to do with this feature/PR, could you please try to revert the `composer.lock` change?

**Comment by mpdude** [comment]
Danke @janopae!

**Pull request merged**